db/migrate/20230427195343_create_event_attendees.rb

```diff
@@ -0,0 +1,10 @@
+class CreateEventAttendees < ActiveRecord::Migration[7.0]
+  def change
+    create_table :event_attendees do |t|
+      t.references :attendee, references: :users, foreign_key: { to_table: :users }, null: false
+      t.references :attended_event, references: :events, foreign_key: { to_table: :events }, null: false
```

Comment:
Given what the table reperesents,  I think there is no need to give a custom name to this column. event_id could work as well